README.md

```diff
@@ -31,17 +39,30 @@ composer create-project fourkitchens/sous-drupal-project PROJECT_NAME --no-inter
     - Review ignored items you may need for your build and remove them
 
 
-### Build project module
+## Working with Emulsify
+The [Emulsify](https://emulsify.info/) theme is installed as part of this project.
 
-Create a new project module
-Generate a custom module at `/web/modules/custom/PROJECT_NAME` using drupal console.
-Follow the documentation for the generate:module command [here](https://hechoendrupal.gitbooks.io/drupal-console/en/commands/generate-module.html)
+Full documentation can be found in the official [Emulsify Documentation](https://docs.emulsify.info/).
+
+### Basic Emulsify Setup
```

Comment:
@mikeethedude Actually this whole section isn't an officially recommended method of setting up the theme with Sous.  When you composer create-project the project it runs the post composer require command: https://github.com/fourkitchens/sous-drupal-project/blob/2.x/composer.json#L108

Which is this script here: https://github.com/fourkitchens/sous-drupal-project/blob/2.x/sous/Starter.php#L19

So until we update this documentation to reflect using the new Emulsify CLI (my rock for next quarter), we should probably leave this one to reflect the composer create-project scripts or if you wanted to run it manually (after a clone) you can do a `lando composer post-create-project-cmd` and it will run the script.

Reply:
Interesting! I didn't catch that. I'll update this too.